### PR TITLE
Add response handling for SubmitJobSync and PerformTsoCommand declarative methods

### PR DESCRIPTION
**Pull request by callbacksin**

### CODE REVIEW

**Review by dnlsv — APPROVED** [review]

File: src/main/kotlin/org/zowe/zdevops/declarative/AbstractZosmfActionWithResult.kt [hunk 1]
```diff
@@ -0,0 +1,109 @@
+/*
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/zdevops/declarative/AbstractZosmfActionWithResult.kt:1]: Change license to:
```
/*
 * Copyright (c) 2024 IBA Group.
 *
 * This program and the accompanying materials are made available under the terms of the
 * Eclipse Public License v2.0 which accompanies this distribution, and is available at
 * https://www.eclipse.org/legal/epl-v20.html
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *   IBA Group
 *   Zowe Community
 */
```

File: src/main/kotlin/org/zowe/zdevops/declarative/jobs/PerformTsoCommandDeclarative.kt [hunk 2]
```diff
@@ -5,70 +5,46 @@
  *
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/zdevops/declarative/jobs/PerformTsoCommandDeclarative.kt:7]: Change license to:
```
/*
 * Copyright (c) 2023-2024 IBA Group.
 *
 * This program and the accompanying materials are made available under the terms of the
 * Eclipse Public License v2.0 which accompanies this distribution, and is available at
 * https://www.eclipse.org/legal/epl-v20.html
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *   IBA Group
 *   Zowe Community
 */
```

File: src/main/kotlin/org/zowe/zdevops/declarative/jobs/SubmitJobSyncStepDeclarative.kt [hunk 3]
```diff
@@ -5,41 +5,46 @@
  *
```
Comment (KUGDev) [inline src/main/kotlin/org/zowe/zdevops/declarative/jobs/SubmitJobSyncStepDeclarative.kt:7]: Change license to:
```
/*
 * Copyright (c) 2022-2024 IBA Group.
 *
 * This program and the accompanying materials are made available under the terms of the
 * Eclipse Public License v2.0 which accompanies this distribution, and is available at
 * https://www.eclipse.org/legal/epl-v20.html
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *   IBA Group
 *   Zowe Community
 */
```

File: src/test/kotlin/org/zowe/zdevops/declarative/jobs/PerformTsoCommandDeclarativeSpec.kt [hunk 4]
```diff
@@ -0,0 +1,119 @@
+/*
```
Comment (KUGDev) [inline src/test/kotlin/org/zowe/zdevops/declarative/jobs/PerformTsoCommandDeclarativeSpec.kt:1]: Change license to:
```
/*
 * Copyright (c) 2024 IBA Group.
 *
 * This program and the accompanying materials are made available under the terms of the
 * Eclipse Public License v2.0 which accompanies this distribution, and is available at
 * https://www.eclipse.org/legal/epl-v20.html
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *   IBA Group
 *   Zowe Community
 */
```

**Review by KUGDev — APPROVED** [review]

**Pull request merged**